httpx/dispatch/http11.py

```diff
@@ -150,6 +166,12 @@ def __init__(
         """
         while True:
             event = self.h11_state.next_event()
+
+            if isinstance(event, h11.Data):
+                logger.debug(f"receive_event event=Data({len(event.data)})")
```

Comment:
Should we format this as something like `Data(<5 bytes>)` to make it clear that it's not a literal?